### PR TITLE
[Fix] Fix wrong init usage in transformer models

### PR DESCRIPTION
**Pull request by Junjun2016**
Fix wrong trunc_normal_init usage

`trunc_normal_init` will init `weight` and `bias` of the input module.
`trunc_nomal_` will init tensor directly.

### CODE REVIEW

**Review by xvjiarui — APPROVED** [review]
Do we have reference PR for this?

**Comment by MengzhangLI** [comment]
@xvjiarui 
Reference PR from MMDetection: https://github.com/open-mmlab/mmdetection/pull/6432.

File: mmseg/models/backbones/mit.py [hunk 1]
```diff
@@ -407,19 +407,14 @@ def init_weights(self):
...
-                    constant_init(m.weight, 1.0)
+                    constant_init(m, val=1.0, bias=0.)
                 elif isinstance(m, nn.Conv2d):
                     fan_out = m.kernel_size[0] * m.kernel_size[
                         1] * m.out_channels
                     fan_out //= m.groups
-                    normal_init(m.weight, 0, math.sqrt(2.0 / fan_out))
-                    if m.bias is not None:
-                        constant_init(m.bias, 0)
+                    normal_init(m, 0, math.sqrt(2.0 / fan_out), 0)
```
Comment (Junjun2016) [inline mmseg/models/backbones/mit.py:417]:
```suggestion
                    normal_init(m, mean=0, std=math.sqrt(2.0 / fan_out), bias=0)
```
```suggestion
                    normal_init(m, 0, math.sqrt(2.0 / fan_out), 0)
```

File: mmseg/models/backbones/swin.py [hunk 2]
```diff
@@ -665,15 +666,10 @@ def init_weights(self):
...
+                trunc_normal_(self.absolute_pos_embed, std=0.02)
             for m in self.modules():
                 if isinstance(m, nn.Linear):
-                    trunc_normal_init(m.weight, std=.02)
-                    if m.bias is not None:
-                        constant_init(m.bias, 0)
-                elif isinstance(m, nn.LayerNorm):
-                    constant_init(m.bias, 0)
-                    constant_init(m.weight, 1.0)
+                    trunc_normal_(m.weight, std=.02)
```
Comment (RangiLyu) [inline mmseg/models/backbones/swin.py:672]:
```suggestion
                    trunc_normal_init(m, std=.02, bias=0.)
                elif isinstance(m, nn.LayerNorm):
                    constant_init(m, 1.0)
```

Comment (Junjun2016) [reply mmseg/models/backbones/swin.py]: resolved, please review it again.

**Review by RangiLyu — APPROVED** [review]

**Pull request merged**